frontend/app/src/test/java/dk/scheduling/schedulingfrontend/CreateTaskPageTest.kt

```diff
@@ -0,0 +1,37 @@
+package dk.scheduling.schedulingfrontend
+
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.TimePickerState
+import dk.scheduling.schedulingfrontend.components.DateRange
+import dk.scheduling.schedulingfrontend.pages.isValidDuration
+import dk.scheduling.schedulingfrontend.pages.isValidInput
+import org.junit.Test
+
+@OptIn(ExperimentalMaterial3Api::class)
+class CreateTaskPageTest {
+    private val validTime = TimePickerState(0, 0, true)
+    private val validTimeMidday = TimePickerState(12, 0, true)
+
+    @Test
+    fun isValidDurationTest() {
```

Comment:
Cant we use a theory with datapoints?

Reply:
No, then we would need to install some new package, which I dont know how to do :)